app/helpers/application_helper.rb

```diff
@@ -1219,7 +1219,10 @@ def allowed_filter_db?(db)
   end
 
   def miq_tab_header(id, active = nil, options = {}, &block)
-    content_tag(:li, :class => "#{options[:class]} #{active == id ? 'active' : ''}", :id => "#{id}_tab") do
+    content_tag(:li,
+                :class     => "#{options[:class]} #{active == id ? 'active' : ''}",
+                :id        => "#{id}_tab",
+                'ng-click' => "changeAuthTab('#{id}');") do
```

Comment:
@h-kataria Do you remember why this change was needed please?

Because now it is putting that `ng-click` in *every* tab, everywhere ... which will break once those other screens are actually angular and try to call it.

I'll move it to `options`, and I'm assuming this is only needed in `app/views/layouts/_multi_auth_credentials_form.html.haml` but .. if you know more.. :)